roles/postgresql/tasks/postgresql_yum6.yml

```diff
@@ -28,9 +28,15 @@
     name: "{{ item }}"
     state: installed
   with_items:
-    - postgresql94-server
-    - postgresql94-contrib
-    - postgresql94-devel
+    # Temp pinning as "/var/lib/pgsql/9.4/data/pg_hba.conf does not exist on postgresql9494-9.4.15-2
```

Comment:
Just a little nitpick but the problem version was `postgresql9494-9.4.15-3`, and the reason for the failure was the `initdb` failure didn't create this file - it should still exist in a proper install.

Again, that context is in my PR which you referenced, so not a big deal, just might be a bit confusing if someone doesn't go hunt that context down.